### PR TITLE
IRIS-3831-update-dependencies

### PR DESCRIPTION
**Pull request by giuliano176**

### CODE REVIEW

**Review by dhavaldodiya — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -33,79 +33,75 @@
...
-		"core-js": "3.19.1",
+		"@zextras/carbonio-design-system": "^1.1.0",
+		"@zextras/carbonio-shell-ui": "^1.1.1",
+		"@zextras/carbonio-ui-preview": "^1.0.0",
+		"core-js": "^3.27.2",
 		"date-arithmetic": "^4.1.0",
-		"formik": "2.1.4",
-		"i18next": "^21.5.4",
-		"immer": "7.0.14",
+		"i18next": "^21.10.0",
```
Comment (nubsthead) [inline package.json:87]: i18next latest is 22.4.9

File: package.json [hunk 2]
```diff
@@ -33,79 +33,75 @@
...
+		"polished": "^4.2.2",
+		"prop-types": "^15.8.1",
+		"react": "^17.0.2",
 		"react-big-calendar": "0.38.1",
-		"react-dom": "17.0.2",
-		"react-i18next": "11.14.3",
-		"react-redux": "7.2.6",
-		"react-router-dom": "^5.3.0",
+		"react-dom": "^17.0.2",
+		"react-i18next": "^11.18.6",
```
Comment (nubsthead) [inline package.json:97]: react-i18next 12.1.5

File: package.json [hunk 3]
```diff
@@ -33,79 +33,75 @@
...
+		"react-redux": "^7.2.9",
+		"react-router-dom": "^5.3.4",
 		"react-widgets": "4.6.1",
 		"react-widgets-moment": "4.0.30",
-		"redux": "^4.1.2",
-		"styled-components": "5.3.6",
-		"zustand": "3.6.9"
+		"redux": "^4.2.1",
+		"styled-components": "^5.3.6",
+		"zustand": "^3.7.2"
```
Comment (nubsthead) [inline package.json]: zustand is 4.3.3

File: package.json [hunk 4]
```diff
@@ -33,79 +33,75 @@
...
+		"pinst": "^3.0.0",
 		"ts-node": "^10.9.1"
 	},
 	"dependencies": {
 		"@emotion/react": "^11.10.4",
 		"@emotion/styled": "^11.10.4",
 		"@mui/icons-material": "^5.10.6",
 		"@mui/material": "^5.10.6",
-		"@reduxjs/toolkit": "^1.6.2",
+		"@reduxjs/toolkit": "~1.7.2",
```
Comment (nubsthead) [inline package.json:80]: @reduxjs/toolkit is 1.9.2

File: package.json [hunk 5]
```diff
@@ -33,79 +33,75 @@
...
+		"prop-types": "^15.8.1",
+		"react": "^17.0.2",
 		"react-big-calendar": "0.38.1",
-		"react-dom": "17.0.2",
-		"react-i18next": "11.14.3",
-		"react-redux": "7.2.6",
-		"react-router-dom": "^5.3.0",
+		"react-dom": "^17.0.2",
+		"react-i18next": "^11.18.6",
+		"react-redux": "^7.2.9",
```
Comment (nubsthead) [inline package.json:98]: react-redux is 8.0.5

**Review by nubsthead — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (giuliano176) [reply package.json:87]: it is shimmed by the shell, version ^21.10.0 is correct

File: package.json (hunk 2 above)
Comment (giuliano176) [reply package.json:97]: it's shimmed by the shell, ^11.18.6 is correct

File: package.json (hunk 4 above)
Comment (giuliano176) [reply package.json:80]: it is shimmed by the shell, ~1.7.2 is correct

File: package.json (hunk 5 above)
Comment (giuliano176) [reply package.json:98]: it is shimmed by the shell, ^7.2.9 is correct

**Review by nubsthead — DISMISSED** [review]

**Review by dhavaldodiya — DISMISSED** [review]

**Review by nubsthead — APPROVED** [review]

**Review by gnekoz — APPROVED** [review]

**Pull request merged**